internal/generator/command/parser.go

```diff
@@ -92,6 +92,35 @@ func (p *parser) loadCommand2(parent *Cmd, base, dir string) *Cmd {
 	return cmd
 }
 
+func (p *parser) loadProvider() *di.Provider {
```

Comment:
Added generator dependency injection to the generated command package, so commands like run and build will have the dependencies they need to build the application generators.

Reply:
This is in addition to dependency injection by the project cli to support custom dependencies in custom cli commands.